external/system-test/playlist.xml

```diff
@@ -13,33 +13,30 @@
 # limitations under the License.
 -->
 <playlist xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../TKG/playlist.xsd">
-	 <test>
-                <testCaseName>sanity_system</testCaseName>
-                <command>docker run --name sanity_system adoptopenjdk-system-test:latest ; \
-                	$(TEST_STATUS); \
-                	docker rm -f sanity_system; \
-                 	docker rmi -f adoptopenjdk-system-test
-                </command>
-                <levels>
-                	<level>sanity</level>
-                </levels>
-                <groups>
-                 	<group>system</group>
-                </groups>
-        </test>
-
-	 <test>
-                <testCaseName>extended_system</testCaseName>
-                <command>docker run --name extended_system adoptopenjdk-system-test:latest _extended.system; \
-					$(TEST_STATUS); \
-                	docker rm -f extended_system; \
-                 	docker rmi -f adoptopenjdk-system-test
-                </command>
-                <levels>
-                        <level>extended</level>
-                </levels>
-                <groups>
-                        <group>system</group>
-                </groups>
-        </test>
+	<test>
+		<testCaseName>sanity_system</testCaseName>
+		<command> $(TEST_ROOT)$(D)external$(D)external.sh --run --tag ${DOCKERIMAGE_TAG} --version ${JDK_VERSION} --impl ${JDK_IMPL} --dir system-test --reportdst $(REPORTDIR) --docker_args $(EXTRA_DOCKER_ARGS) ; \
+		$(TEST_STATUS); \
+		$(TEST_ROOT)$(D)external$(D)external.sh --clean --tag ${DOCKERIMAGE_TAG} --version ${JDK_VERSION} --impl ${JDK_IMPL} --dir system-test
+		</command>
+        <levels>
+            <level>sanity</level>
+        </levels>
+        <groups>
+            <group>system</group>
+        </groups>
+	</test>
+    <test>
+		<testCaseName>extended_system</testCaseName>
+		<command> $(TEST_ROOT)$(D)external$(D)external.sh --run --tag ${DOCKERIMAGE_TAG} --version ${JDK_VERSION} --impl ${JDK_IMPL} --dir system-test --testtarget _extended.system --reportdst $(REPORTDIR) --docker_args $(EXTRA_DOCKER_ARGS) ; \
```

Comment:
Need to add --testtarget _extended.system.regular

See https://github.com/james-crowley/openjdk-tests/blob/convert_external_tests/external/functional-test/playlist.xml#L31 for similar thing.

Reply:
@smlambert Is the test target `_extended.system` or `_extended.system.regular`? On the old extended system test the target was just `_extended.system`.

Reply:
Ah, then use _extended.system

Note that:
_extended.system = _extended.system.regular + _extended.system.native

It will work, because we build the system test natives when we compile system test material, which is different than the functional group, where we do not build the functional test native images as part of test compilation, but have to pass in the prebuilt ones, so we restrict functionals to use just the non-native/regular tests).